playground/nextjs/pages/_app.tsx

```diff
@@ -58,7 +58,11 @@ const AppBar = (props: AppBarProps) => {
       }}
     >
       <UnstableOrganizationSwitcher afterCreateOrganizationUrl={'https://www.google.com'} />
-      <UnstableOrganizationSwitcher hidePersonal />
+      <UnstableOrganizationSwitcher
+        hidePersonal
+        afterLeaveOrganizationUrl={'https://www.google.com'}
```

Comment:
:upside_down_face: Let's replace 'https://www.google.com' with 'https://clerk.dev' :wink:

Reply:
You're right :) Done!